Heimdallr/iOSHeimdallr.swift

```diff
@@ -0,0 +1,131 @@
+import Foundation
+import Result
+import UIKit
+
+public extension Heimdallr {
+
+    /// Requests an access token with the authorization code grant.
+    ///
+    /// **Note:** The completion closure may be invoked on any thread.
+    ///
+    /// - Parameters:
+    ///   - url: The authorization code link.
+    ///   - redirectURI: The redirect URI.
+    ///   - scope: The authorization scope.
+    ///   - completion: A cllback to invoke when the request completed.
+    public func requestAccessToken(authorizationCodeURL url: URL,
```

Comment:
Why is it named `requestAccessToken` and calls `requestAuthorizationCode` internally?

Reply:
The authorization code grant has two steps:

1. Request an authorization
2. Exchange it for an access token